README.md

```diff
@@ -5,6 +5,12 @@ One second to read GitHub code with VS Code.
 
 [Chrome Extension](https://chrome.google.com/webstore/detail/github1s/lodjfmkfbfkpdhnhkcdcoonghhghbkhe/)
 
+Or Save as a bookmarklet (GitHub markdown doesn't allow js links, just copy it into a bookmark) 
+
+```
+javascript: window.location.href = window.location.href.replace('github.com', 'github1s.com')
```

Comment:
Could you please use this syntax to make it draggable? https://joeldare.com/creating_a_drag_and_drop_bookmarklet.html#providing-a-bookmarklet-in-markdown

Reply:
I tried and GitHub wont render it as a link. Its a security feature for javascript urls

Reply:
Thanks for sharing this.